### PR TITLE
output org, quota and usage info

### PR DESCRIPTION
**Pull request by spgreenberg**
## Changes proposed in this pull request:

Outputs org and query usage in CSV format for the BU team. For example: https://docs.google.com/spreadsheets/d/1yM8-oNL76_Jtx7BLUi4SPdJcg8mAWwKNBv-GqNnShWA/edit?usp=sharing

## security considerations

This outputs org names, quotas, and GUIDs. Treat it accordingly.

### CODE REVIEW

**Review by mheadd — APPROVED** [review]
Nice! This will be very useful.

**Review by ccostino — APPROVED** [review]
Thank you, @spgreenberg!  Nice and straightforward, and easy to pipe or copy the output into something else to parse the data as needed.

File: audit/get-sandbox-user-emails.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+orgs=($(cf orgs | grep '^sandbox'))
+
+echo "Username, Org GUID"
+
+# Loop through the array and look up quota details.
+for org in "${orgs[@]}"
+do
+    guid=$(cf org $org --guid)
+
+    emails=($(cf curl "/v2/organizations/$guid/users" | jq '.resources | .[].entity | .username'))
```
Comment (ccostino) [inline audit/get-sandbox-user-emails.sh:13]: We might want to shift this to the `v3` API call - that may change the response output though, so double check that it's what you expect!

**Review by ccostino — COMMENTED** [review]
Thanks for the addition, @mheadd!  We may want to swap to the `v3` API, though; the response is likely different from the `v2` response so the `jq` processing may need some tweaking.

File: audit/get-sandbox-user-emails.sh (hunk 1 above)
Comment (mheadd) [reply audit/get-sandbox-user-emails.sh:13]: That's fair. I'll merge this, but will create an issue to update the script for the newer version of the CF API.

Comment (ccostino) [reply audit/get-sandbox-user-emails.sh:13]: Sounds good, thanks @mheadd!

**Pull request merged**